SCons/Tool/DCommon.xml

```diff
@@ -350,6 +350,37 @@ See also &cv-link-DLINKFLAGS; for linking static objects.
 </summary>
 </cvar>
 
+<cvar name="DIFILESUFFIX">
+<summary>
+<para>
+Suffix of d include files default is .di
+</para>
+</summary>
+</cvar>
+
+<cvar name="DINTFDIRKEY">
```

Comment:
DINTFDIRKEY ? seems an odd variable name.
Can you explain?

Reply:
Can't explain, wrote this a long time ago, merging now because I found that my code had been broken.

Reply:
DINTFDIR ? Don't need the key

Reply:
I'm presuming it's for this:
```
       -Hd dir
           Same as -H, but writes interface files to directory dir.  This
           option can be used with -Hf file to independently set the output
           file and directory path.
```

Reply:
`DIFILEDIR`?

Reply:
Yes it's for -Hd dir. The file specific -Hf version should probably have a SCons implementation, but I haven't used it.
DIFILEDIR also good.

Reply:
How about `DI_FILE_DIR` ?  @mwichmann 
I know we have a long history of VERYLONGVARIABLESWHICHAREHARDTOREAD... but I think there's no benefit in continuing that?